public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -884,6 +885,41 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
     });
   }
 
+  public getDashboardPanels = () => {
```

Comment:
this looks like it duplicates dashboardSceneGraph.getVizPanels

Reply:
ahh nice, I will use this one instead